### PR TITLE
Push avg and median

### PR DESCRIPTION
**Pull request by hightoxicity**

### CODE REVIEW

File: unbound_exporter.go [hunk 1]
```diff
@@ -218,6 +218,18 @@ var (
 			prometheus.CounterValue,
 			nil,
 			"^unwanted\\.replies$"),
+		newUnboundMetric(
+			"recursion_time_avg",
```
Comment (EdSchouten) [inline unbound_exporter.go]: The Prometheus naming guidelines suggest that the unit of what's encoded should be included in the name, e.g. `recursion_time_seconds_avg`.

**Comment by EdSchouten** [comment]
Ah, it looks like this is a metric that was added to versions of Unbound newer than the ones we still run. Thanks!

What I dislike is that these kinds of metrics (exposing the median and average) is completely orthogonal to Prometheus' best practices. This should just expose a histogram, containing buckets, count and sum. That said, there's little we can do about that, right? :-/

**Comment by hightoxicity** [comment]
I think the histo represents all response times including in-cache entries (but not very clear in unbound doc), the precomputed mean + avg exposed by unbound is quite precise and concern only outgoing calls (the ones who need recursion) response time.

File: unbound_exporter.go (hunk 1 above)
Comment (hightoxicity) [reply unbound_exporter.go]: that's true! And better!

**Comment by hightoxicity** [comment]
Thanks guy, I updated metric names.

**Pull request merged**